template/runner/graph_classification_iamdb/graph_classification_iamdb.py

```diff
@@ -0,0 +1,179 @@
+import os
+import logging
+
+# Gale
+from template.runner.base import BaseRunner
+
+# Delegated
+from .setup import GraphClassificationSetup
+from .train import GraphClassificationTrain
+from .evaluate import GraphClassificationEvaluate
+
+
+class GraphClassificationIamdb(BaseRunner):
```

Comment:
is this runner peculiar to the iamdb ? Can it be a general GNN runner?

Reply:
It expects the same data structure as there is in the IAMDB database, with gxl files with the graphs and cxl files for the dataset splits and class labels. But I can definitely rename it to GraphClassificationGxl. Does that make more sense?

Reply:
If it is a common standard it could make sense but i have no experience in GNN datasets so you will have to make the call